src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+export PATH=/root/bin:$PATH
+mkdir /root/bin
+{% if ceph_dev_folder is defined %}
+  cp /mnt/{{ ceph_dev_folder }}/src/cephadm/cephadm /root/bin/cephadm
+{% else %}
+  cd /root/bin
+  curl --silent --remote-name --location https://raw.githubusercontent.com/ceph/ceph/master/src/cephadm/cephadm
```

Comment:
I think this should point to octopus, but its not a big deal since in the script the `ceph_dev_folder` will always be defined
```suggestion
  curl --silent --remote-name --location https://raw.githubusercontent.com/ceph/ceph/octopus/src/cephadm/cephadm

Reply:
Done.